packages/components/tests/dummy/app/templates/components/gutters.hbs

```diff
@@ -0,0 +1,186 @@
+{{page-title "Gutters Component"}}
+
+<h2 class="dummy-h2">Gutters</h2>
+
+<section>
+  <h3 class="dummy-h3" id="overview">
+    <a href="#overview" class="dummy-link-section">§</a>
+    Overview
+  </h3>
+
+  <p class="dummy-paragraph">
+    Use to add consistent vertical or horizontal “gutter” spacing between child elements.
+  </p>
+  <p class="dummy-paragraph">
+    (Similar to
+    <a href="https://every-layout.dev/layouts/stack/" target="_blank" rel="noopener noreferrer">“The Stack” pattern from
+      “Every Layout”</a>
+    and
+    <a href="/components/button-set/">"Hds::ButtonSet"</a>.)
+  </p>
+</section>
+
+<section>
+  <h3 class="dummy-h3" id="component-api">
+    <a href="#component-api" class="dummy-link-section">§</a>
+    Component API
+  </h3>
+
+  <p class="dummy-paragraph" id="component-api-gutters">
+    Here is the API for the component:
+  </p>
+  <dl class="dummy-component-props" aria-labelledby="component-api-gutters">
+    <dt>spacing</dt>
+    <dd>
+      <p>Choose from either a preset spacing scale or else set a custom css unit spacing size.</p>
+      <p>
+        <strong class="dummy-paragraph">Spacing scale:</strong>
+        "xxs", "xs", "sm", "md", "lg", "xl", "xxl"<br />
+        (Spacing scale values map to a
+        <a
+          href="https://www.figma.com/file/oQsMzMMnynfPWpMEt91OpH/HDS-Product---Foundations?node-id=2008%3A3"
+          target="_blank"
+          rel="noopener noreferrer"
+        >loose spacing scale in HDS</a>.)
+      </p>
+      <p>
+        <strong class="dummy-paragraph">Supported CSS units:</strong>
+        "px", "rem", "em", "%"<br />
+        (Negative values are not supported.)
+      </p>
+      <p class="dummy-paragraph">Default spacing size = "md"</p>
+    </dd>
+
+    <dt>direction</dt>
+    <dd>
+      <p>vertical, horizontal (Default, if any, to be determined)</p>
+    </dd>
+
+    <dt>TOOD: tag/inline/block</dt>
+    <dd>
+      <p>Needs more thought. (To prevent invalid markup, users would be able to choose between using a div tag (the
+        default) or a span tag.)</p>
+    </dd>
```

Comment:
I would like to give users the option to use a span tag instead of the default div tag for the component but not sure what to call the argument.

Reply:
ok, let's discuss what options do we have.

Reply:
I'm now thinking I will call the argument: "isSpan". The default would be to use a div tag. I'll work on this...

Reply:
Added new isSpan argument